docs/pages/management/admin/daemon.mdx

```diff
@@ -33,89 +31,101 @@ $ readlink /sbin/init
-you how to run Teleport as a daemon:
+Teleport requires a configuration YAML file to be created and configured in order to run. After initial installation, no
+configuration file exists until we create it. This guide will use a minimal Teleport configuration based on defaults, however in production environments we
+recommend reviewing configuration file options to better suit your needs.
+
+To create the default Teleport configuration file, enter the following command:
 
 ```code
 $ sudo teleport configure -o file
 Wrote config to file "/etc/teleport.yaml". Now you can start the server. Happy Teleporting!
 ```
 
-## Step 2/3. Create a systemd unit file
+## Step 2/3. Create and configure a systemd service
 
-Copy the recommended Teleport unit file for `systemd` below and paste the
-contents into a file called `/usr/lib/systemd/system/teleport.service`:
+Once you've installed the `teleport` binary and created a Teleport configuration file, let's create and configure the Teleport daemon.
 
-```systemd
-[Unit]
-Description=Teleport SSH Service
-After=network.target
+Systemd is designed to use **unit files**, which are the files used to define and configure
+any service daemon on systemd, in this case the `teleport` service. The rest of this guide will focus on creating and configuring the `teleport` service daemon depending on your preferred installation method, either a package manager, or from source.
 
-[Service]
-Type=simple
-Restart=on-failure
-ExecStart=/usr/local/bin/teleport start --config=/etc/teleport.yaml --pid-file=/run/teleport.pid
-ExecReload=/bin/kill -HUP $MAINPID
-PIDFile=/run/teleport.pid
+<Tabs>
+  <TabItem label="Package Manager Installation">
 
-[Install]
-WantedBy=multi-user.target
-```
+  We have included a default `teleport` daemon configuration for packages installed with DEM or RPM package managers. To check that this daemon was installed correctly,
+  enter the following command:
 
-<Notice type="tip">
+  ```code
+  $ sudo systemctl status teleport
+  ```
 
-If `/usr/lib/systemd/system/` does not exist, consult the list of
-[unit file load paths](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Unit%20File%20Load%20Path)
-for other supported paths.
+  You will see output similar to the following, including the file path (`/lib/systemd/system/teleport.service`) that contains the unit file for the systemd configuration being applied:
 
-</Notice>
+  ```code
+  ● teleport.service - Teleport SSH Service
+       Loaded: loaded (/lib/systemd/system/teleport.service; disabled; vendor preset: enabled)
+       Active: inactive (dead)
+  ```
 
-Enable the unit so systemd can place it in its dependency tree:
+  Enter the following command to enable the systemd `teleport` daemon. Enabling the daemon using `systemctl` allows systemd to place the daemon in its dependency tree, ensuring that the teleport service will be automatically restarted on any potential reboot:
```

Comment:
```suggestion
  Enter the following command to enable the systemd `teleport` daemon. Enabling the daemon using `systemctl` allows systemd to place the daemon in its dependency tree, ensuring that the Teleport service will be automatically restarted on any potential reboot:
```

If you're referring to the product in general and not specifically to the `teleport` program, I think this should be capitalized, right?